e2e/wdio/core/tests/menu.e2e-spec.ts

```diff
@@ -0,0 +1,120 @@
+import { MenuPo } from '../pages/menu.po';
+import { checkElementScreenshot,
+    click, 
+    mouseHoverElement, 
+    saveElementScreenshot, 
+    addIsActiveClass,
+    focusElement, 
+    getElementArrayLength,
+    pause,
+    waitForElDisplayed,
+    elementDisplayed,
+    doesItExist} from '../../driver/wdio';
+
+describe('Menu tests', function() {
+    const menuPage = new MenuPo();
+    const { menuButtonsArr, 
+        btnMobileMenu, 
+        menuItemsArr, 
+        dialogMobileMenu,
+        closeDialogMobileMenu,
+        dialogMenuAddonArr,
+        dialogMenuItemsArr,
+        dialogBtnBack } = menuPage;
+
+    beforeAll(() => {
+        menuPage.open();                              
+    }, 1);
+
```

Comment:
please add checks for:
- menu items clickable
- menu items with icons have icons
- menu opens on click, closes on 2nd click
- ability to navigate through menu items with tab nav and arrow keys
- that compact menu has compact attribute
- that the active path on the 'menu with submenu' example shows correct item